### ISSUE TITLE
strange nullpointer when trying to resume

### ISSUE THREAD

**Issue by MCOfficer** [EndlessSkyCommunity/EndlessSky-Discord-Bot#101]
not really reproducable
```
Caused by: java.lang.NullPointerException: null
        at bot.AudioPlayerVoteHandler.vote(AudioPlayerVoteHandler.java:43)
        at bot.PlayerControl.vote(PlayerControl.java:803)
        at bot.PlayerControl.onResumeCommand(PlayerControl.java:511)
```

**Comment by MCOfficer** [comment]
this was not meant to be closed ^

**Comment by tehhowch** [comment]
It's possible this is the reconnect bug. I don't believe removing any "null" members will avoid this nullpointer when resuming (since no properties of the voters is actually done, beyond passing them around where they are checked to be in a given list).

**Comment by tehhowch** [comment]
possibly addressed with #114
